examples/acm-terraform-blog-part3/terraform/main.tf

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_gke_hub_membership" "membership" {
+  provider      = google-beta
+  membership_id = "membership-hub"
```

Comment:
This should probably be derived from the cluster name (since each cluster needs a unique membership ID).

Reply:
Changed to `membership_id = "membership-hub-${module.gke.name}"` for all parts - 1, 2 and 3.